librz/bin/format/pe/pe_rsrc.c

```diff
@@ -338,7 +338,7 @@ static String *Pe_r_bin_pe_parse_string(RzBinPEObj *bin, PE_DWord *curAddr) {
 		if (*curAddr > bin->size || *curAddr + sizeof(ut16) > bin->size) {
 			goto out_error;
 		}
-		if (!rz_buf_read_le16_at(bin->b, *curAddr, &utf16_char)) {
+		if (rz_buf_read_at(bin->b, *curAddr, (ut8 *)&utf16_char, sizeof(ut16)) != sizeof(ut16)) {
```

Comment:
Note, that technically UTF16 could be also BE. We need to check if Windows PEs for big-endian platforms still used UTF16LE variant.

Reply:
no, this is correct because the conversion is done by `rz_str_utf16_to_utf8`